hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -288,6 +292,8 @@ public BucketCache(String ioEngineName, long capacity, int blockSize, int[] buck
     this.singleFactor = conf.getFloat(SINGLE_FACTOR_CONFIG_NAME, DEFAULT_SINGLE_FACTOR);
     this.multiFactor = conf.getFloat(MULTI_FACTOR_CONFIG_NAME, DEFAULT_MULTI_FACTOR);
     this.memoryFactor = conf.getFloat(MEMORY_FACTOR_CONFIG_NAME, DEFAULT_MEMORY_FACTOR);
+    this.queueAdditionWaitTime = conf.getLong(QUEUE_ADDITION_WAIT_TIME,
+      DEFAULT_QUEUE_ADDITION_WAIT_TIME);
```

Comment:
Any cons on setting a default value > 0?

Reply:
By default don't want to add wait time because memory based offheap will be faster. So it can be completely depending up on the use case like if the file based offheap used eg. cloud ephemeral storages then configuring wait time would be helpful.